pkg/capr/planner/planner.go

```diff
@@ -1105,6 +1093,18 @@ func (p *Planner) generatePlanWithConfigFiles(controlPlane *rkev1.RKEControlPlan
 
 		nodePlan.Files = append(nodePlan.Files, idempotentScriptFile)
 
+		bootstrapManifests, err := p.retrievalFunctions.GetBootstrapManifests(controlPlane)
+		if err != nil {
+			return nodePlan, config, joinedServer, err
+		}
+
+		if len(bootstrapManifests) > 0 {
+			logrus.Debugf("[planner] adding pre-bootstrap manifests")
+			nodePlan.Files = append(nodePlan.Files, bootstrapManifests...)
+		} else {
+			nodePlan, err = p.addManifests(nodePlan, controlPlane, entry)
```

Comment:
@thatmidwesterncoder , is there a reason `p.addManifests` moved to under `else` section here while before the change it was outside of `if` block?

Reply:
This is no longer relevant following the latest push that changes code a bit.